modules/datastore/src/Storage/DatabaseTable.php

```diff
@@ -181,15 +181,15 @@ private function buildTableSchema($tableName, $fieldsInfo) {
    *
    * @param string $type
    *   Type returned from the describe query.
-   * @param string|null $extra
-   *   MySQL "Extra" property for column.
+   * @param mixed $extra
+   *   Additional information for column.
    *
    * @return string
    *   Fritionless Table Schema compatible type.
    *
    * @see https://specs.frictionlessdata.io/table-schema
    */
-  private function translateType(string $type, $extra = NULL) {
+  public function translateType(string $type, $extra = NULL) {
```

Comment:
Is there intention to use this method elsewhere? If not, we should be safe to just use protected here.
```suggestion
  protected function translateType(string $type, $extra = NULL) {
```